presto-orc/src/main/java/com/facebook/presto/orc/metadata/Stream.java

```diff
@@ -130,4 +132,28 @@ public Stream withOffset(long offset)
                 this.sequence,
                 Optional.of(offset));
     }
+
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Stream stream = (Stream) o;
+        return column == stream.column
```

Comment:
Is column always properly populated? I vaguely remembered it was not used

Reply:
It's actually a misnamed nodeId, streams in presto-orc don't have the column.